### ISSUE TITLE
Accessibility: attributes on modal dialogue box

### ISSUE THREAD

**Issue by laurelfulford** [Automattic/republication-tracker-tool#135]
This came out of an accessibility report done on the site of one of Newspack's publishers:

The modal is missing a couple of recommended attributes for screen readers -- the fix is making sure they're present, whether or not the modal is opened or focused on -- from the feedback: 

"Ensure the attributes `role="dialog"` and `aria-modal="true"` are present on the wrapper element of this modal dialog. Do not add these attributes dynamically after the user's focus has been moved to it or into it."

Resource: https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
